src/pages/MenuDetail.vue

```diff
@@ -105,5 +105,7 @@ export default {
   },
 }
 </script>
-<style>
+<style scoped lang="stylus">
+.text-modifier
+  color #616161
```

Comment:
color #616161 possible to use color $variable ?

Reply:
I only use setting color variable when it is used least twice times in app.
Example : facebook color (it is used ast Register and Login page)

Reply:
yes it should be like that with other css styling properies, but with color, we try to stick around with quasar pre-defined name like primary,secondary,.... so that we could control the app color theme at one place, for #616161, can we just fall back to use primary/secondary... ? unless #616161 is a very important color for the page.